### PR TITLE
Improve MessageListParams interface in ChannelProvider

### PR DESCRIPTION
**Pull request by HoonBaek**
fix from https://github.com/sendbird/sendbird-uikit-react/pull/980

thanks to @tylerhammer

### CODE REVIEW

File: src/modules/Channel/context/ChannelProvider.tsx [hunk 1]
```diff
@@ -51,23 +52,13 @@ import { useHandleChannelPubsubEvents } from './hooks/useHandleChannelPubsubEven
...
-  customTypes?: Array<string>,
-  senderUserIds?: Array<string>,
-};
-
+export interface MessageListParams extends Partial<SDKMessageListParams> { // make `prevResultSize` and `nextResultSize` to optional
+  /**
+   * @deprecated
+   * It won't work even if you activate this props
+   */
+  reverse?: boolean; // TODO: Deprecate this props, because it might not work
```
Comment (bang9) [inline src/modules/Channel/context/ChannelProvider.tsx:60]: todo?

**Review by bang9 — APPROVED** [review]

File: src/modules/Channel/context/ChannelProvider.tsx (hunk 1 above)
Comment (HoonBaek) [reply src/modules/Channel/context/ChannelProvider.tsx:60]: I thought we would delete this prop from the interface in the future

Comment (bang9) [reply src/modules/Channel/context/ChannelProvider.tsx:60]: "Deprecated" indicates that the API is no longer in use and may be removed in the future.
After being deprecated, the process of removal through a major update is termed "Removed"

Comment (bang9) [reply src/modules/Channel/context/ChannelProvider.tsx:60]: You have already left a **deprecated** mark~!

Comment (HoonBaek) [reply src/modules/Channel/context/ChannelProvider.tsx:60]: aha;; let me remove it in the other place, thank you!

**Pull request merged**